### DISCUSSION TITLE
Can't import a Tiled-created map

### DISCUSSION THREAD

**Discussion by jumpjack** [photonstorm/phaser#6288] [General]
I am trying to import [this map](https://github.com/jumpjack/isometric-game-js-test/blob/master/Jerry-isometric-rpg/data/map/map-level0-embedded-v02.tmx) in several isometric engines, Phaser3 included, but with no success, can anybody help?

Main issue is that I can't find enough documentation or examples on isometric support; I know that previously a plugin was needed, and there are several tutorial on it, but I also know that Phaser supports isometric maps natively since 3.5.0, and I don't want to use an old feature such as the isometric plugin.

I just would like to have a very basic implementation, with a static sprite which moves around controlled by WASD or cursor keys and collides properly, then I will implement the game logic by myself..

**Comment by pacifiquem** [comment]
here are links to fully and well explained Documentation of several isometric support
[Link 1](https://phaser.discourse.group/t/phaser-3-5-isometric-demo-how-to-continue/8543), [Link 2](https://phaser.discourse.group/t/isometric-support/558), [Link 3](https://tnodes.medium.com/creating-an-isometric-view-in-phaser-3-fada95927835) .
